bool success   # true if the ground has been successfully found
float32 X      # ground location coordinates
float32 Y
float32 Z
string frame   # reference frame that ground coordinates refer to
